### PR TITLE
optional logo in quick-dapp

### LINKED ISSUE #5291: QuickDapp Change UI for optional logo input
Currently if no logo is chosen then the user thinks that it is required to use the Remix logo.  Only when you hover on the Remix logo do you see the instructions.  But if they don't have a logo - they are stuck.

Right now the UI looks like this:

image

A rough sketch of the UI should be more like this:

image

### PR DESCRIPTION
**Pull request by drafish**
fix #5291

### CODE REVIEW

File: apps/quick-dapp/src/actions/index.ts [hunk 1]
```diff
@@ -192,7 +193,9 @@ export const deploy = async (payload: any, callback: any) => {
     files[`dir/${path}`] = resp.data;
   }
 
-  files['dir/assets/logo.png'] = logo
+  if (logo) {
```
Comment (yann300) [inline apps/quick-dapp/src/actions/index.ts:196]: there is no default case anymore?

**Comment by ryestew** [comment]
The logo for uploading a logo is too large:
image

It also does not say that it is optional.

File: apps/quick-dapp/src/actions/index.ts (hunk 1 above)
Comment (drafish) [reply apps/quick-dapp/src/actions/index.ts:196]: That's it. No more default logo. @yann300

File: apps/quick-dapp/src/actions/index.ts [hunk 2]
```diff
@@ -353,7 +356,7 @@ export const initInstance = async ({
       natSpec,
       solcVersion: getVersion(solcVersion),
       ...lowLevel,
-      logo: logo.data,
+      // logo: logo.data,
```
Comment (LianaHus) [inline apps/quick-dapp/src/actions/index.ts:359]: please remove if not needed

File: apps/quick-dapp/src/components/DeployPanel/index.tsx [hunk 3]
```diff
@@ -51,6 +51,8 @@ function DeployPanel(): JSX.Element {
     <div className="col-3 d-inline-block">
       <h3 className="mb-3" data-id="quick-dapp-admin">QuickDapp <FormattedMessage id="quickDapp.admin" /></h3>
       <Button
+        size="sm"
```
Comment (LianaHus) [inline apps/quick-dapp/src/components/DeployPanel/index.tsx:54]: please use "btn btn-sm" instead of custom style

**Review by LianaHus — CHANGES_REQUESTED** [review]

**Comment by Aniket-Engg** [comment]
closing as #5376 is merged

**Pull request closed without merging**